packages/playback/src/lib/service-locator.ts

```diff
@@ -18,6 +18,7 @@ import { EventEmitter } from './utils/event-emitter';
 import { NetworkManager } from './network/network-manager';
 import { PlayerEventType } from './consts/events';
 import { PipelineLoaderFactoryStorage } from './utils/pipeline-loader-factory-storage';
+import { FullPlaylistParser } from '@videojs/hls-parser';
```

Comment:
We don't want it to be included in the player's default bundle. It should only be a part of the HLs pipeline. And the users should be able to set a custom factory in the HLS pipeline only